cookiecutter/generate.py

```diff
@@ -263,17 +267,10 @@ def generate_files(repo_dir, context=None, output_dir='.',
 
     unrendered_dir = os.path.split(template_dir)[1]
     ensure_dir_is_templated(unrendered_dir)
-    env = StrictEnvironment(
-        context=context,
-        keep_trailing_newline=True,
-    )
+    env = StrictEnvironment(context=context, keep_trailing_newline=True,)
```

Comment:
Trailing commas in such places should be removed

Reply:
@inglesp I tried to remove the but some of them are added back by black itself.

Reply:
I think you meant @insspb...